### ISSUE TITLE
New champions have not been updated from pull_data

### ISSUE THREAD

**Issue by feimaomiao** [meraki-analytics/role-identification#14]
pull_data only pulls champions up to version 11.12, newer champions like Vex and Zeri has not been added.

**Comment by knowledge1st** [comment]
This still happens, any idea how to fix or create one by my own? Thanks in advance.

**Comment by jjmaldonis** [comment]
Yes this is an issue. We are hoping someone will help us by contributing to the championrates file in this repo: https://www.paypal.me/merakianalytics which supports the role-identification code.
